File: ConsoleApp.FSharp/FizzBuzz.fs
namespace ConsoleApp.FSharp

open type System.Console

type FizzBuzz =
    static member Run fromNum toNum =
        for i = fromNum to toNum do
            let mutable result = ""
            if i % 3 = 0 then result <- result + "Fizz"
            if i % 5 = 0 then result <- result + "Buzz"
            let message = if result = "" then i.ToString() else result
            WriteLine message

File: ConsoleApp.CSharp/FizzBuzz.cs
namespace ConsoleApp.CSharp;

using static Console;

public static class FizzBuzz
{
    public static void Run()
    {
        for (var i = 1; i <= 100; i++)
        {
            var result = "";
        
            if (i % 3 == 0) result += "Fizz";
            if (i % 5 == 0) result += "Buzz";

            WriteLine(result == "" ? i : result);
        }
    }
}